samples/multicore/hello_world/power_management_app.overlay

```diff
@@ -1,24 +1,25 @@
 /*
  * Copyright (c) 2024 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: Apache-2.0
  */
 
- /delete-node/ &cpuppr_ram3x_region;
- /delete-node/ &cpuapp_cpuppr_ipc;
+/ {
+	power-states {
+		idle: idle {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-idle";
+			min-residency-us = <100000>;
+		};
 
- / {
-         reserved-memory {
-                 cpuppr_ram3x_region: memory@2fc00000 {
-                         compatible = "nordic,owned-memory";
-                         reg = < 0x2fc00000 0x10000 >;
-                         status = "okay";
-                         perm-read;
-                         perm-write;
-                         perm-execute;
-                         #address-cells = < 0x1 >;
-                         #size-cells = < 0x1 >;
-                         ranges = < 0x0 0x2fc00000 0x10000 >;
-                 };
-         };
- };
+		s2ram: s2ram {
```

Comment:
The name might be a bit unfortunate. If someone doesn't know it means suspend to ram he might think it is some strange sram node. Maybe sus2ram to differentiate it from sram?

Reply:
S2RAM is a common abbreviation used all over computers (ok, mostly in Linux context :) ). I've never seen sus2ram.